fuel/transformers/__init__.py

```diff
@@ -619,3 +619,33 @@ def get_data(self, request=None):
             raise ValueError
         data = next(self.child_epoch_iterator)
         return data
+
+
+class FilterSources(Transformer):
+    """Selects a subset of the stream sources.
```

Comment:
I'd add a paragraph explaining that the data stream's source order is maintained.